corehq/apps/app_manager/tests/test_ccz.py

```diff
@@ -0,0 +1,81 @@
+# coding: utf-8
```

Comment:
TIL how python decides on encoding: https://docs.python.org/2/reference/lexical_analysis.html#encoding-declarations

I was going to say we should consolidate on the recommended approach, but I think for python 3 all files are assumed to be utf-8 so its won't be required